### PR TITLE
Wiki to hub link update.

### PR DESCRIPTION
**Pull request by dannon**
Unfortunately there are quite a few of these.  I updated a couple and pretty much immediately ran into a missing page, so I'm going to add those as I hit them.

As mentioned in one of the commits, this logic needs to be consolidated in a lot of places.

This may be WIP for a while.

### CODE REVIEW

**Comment by nsoranzo** [comment]
`doc/source/` also has a bunch of them.

**Comment by dannon** [comment]
Yep, there are quite a few.  I'm grepping through incrementally.

**Comment by martenson** [comment]
Redirects are in place so I believe we do not need to commit to a big cleanup.

**Comment by dannon** [comment]
I agree that because of the redirects it's not particularly important to get this out the door right away, but we should still update URLs where they are incorrect (or, let's say, can be improved to be more accurate as the current URL), and I'll do so in this PR.

**Comment by nsoranzo** [comment]
@dannon If you fix the merge conflict and remove the WIP label, I'll review and eventually merge this.

**Comment by dannon** [comment]
@nsoranzo Yep.  Going to make one quick pass over it when we're off the call and will take it out of WIP today.

**Comment by dannon** [comment]
(back in WIP for a sec -- merge looks like it had an issue)

edit: fixed with 0dd75e6

**Comment by nsoranzo** [comment]
I suppose the qunit build failure is legit?

**Comment by dannon** [comment]
@nsoranzo Yep, could be, I'll look at it.

**Comment by dannon** [comment]
Yep, just need to update a test case, one sec.

**Comment by nsoranzo** [comment]
Thanks, I thought that was the case. Let's do the other tests too when Jenkins is back.

**Comment by jmchilton** [comment]
@galaxybot test this

File: CITATION [hunk 1]
```diff
@@ -16,7 +16,7 @@ following publications:
   a platform for interactive large-scale genome analysis." 
   Genome Research. 2005 Oct; 15(10):1451-5.
 
-See also: http://wiki.galaxyproject.org/CitingGalaxy
+See also: http://galaxyproject.org/citing-galaxy
```
Comment (nsoranzo) [inline CITATION]: https

File: client/galaxy/scripts/mvc/library/library-folderlist-view.js [hunk 2]
```diff
@@ -386,7 +386,7 @@ var FolderListView = Backbone.View.extend({
       '</table>',
       '<div class="empty-folder-message" style="display:none;">',
           'This folder is either empty or you do not have proper access permissions to see the contents. If you expected something to show up',
-          ' please consult the <a href="https://wiki.galaxyproject.org/Admin/DataLibraries/LibrarySecurity" target="_blank">library security wikipage</a>',
+          ' please consult the <a href="https://galaxyproject.org/admin/data-libraries/library-security" target="_blank">library security wikipage</a>',
```
Comment (nsoranzo) [inline client/galaxy/scripts/mvc/library/library-folderlist-view.js]: https://galaxyproject.org/data-libraries/#permissions

File: client/galaxy/scripts/mvc/workflow/workflow-forms.js [hunk 3]
```diff
@@ -151,7 +151,7 @@ define( [ 'utils/utils', 'mvc/form/form-view', 'mvc/tool/tool-form-base' ], func
                     type        : 'text',
                     value       : '',
                     ignore      : '',
-                    help        : 'This action will rename the output dataset. Click <a href="https://wiki.galaxyproject.org/Learn/AdvancedWorkflow/Variables">here</a> for more information. Valid inputs are: <strong>' + input_terminal_names.join(', ') + '</strong>.'
+                    help        : 'This action will rename the output dataset. Click <a href="https://galaxyproject.org/learn/advanced-workflow/variables-edit/">here</a> for more information. Valid inputs are: <strong>' + input_terminal_names.join(', ') + '</strong>.'
```
Comment (nsoranzo) [inline client/galaxy/scripts/mvc/workflow/workflow-forms.js]: https://galaxyproject.org/learn/advanced-workflow/variables/

File: config/galaxy.ini.sample [hunk 4]
```diff
@@ -615,19 +616,19 @@ nglims_config_file = tool-data/nglims.yaml
...
-#support_url = https://wiki.galaxyproject.org/Support
+#support_url = https://galaxyproject.org/support
 
 # The URL linked by the "How to Cite Galaxy" link in the "Help" menu.
-#citation_url = https://wiki.galaxyproject.org/CitingGalaxy
+#citation_url = https://galaxyproject.org/citing-galaxy
 
 # The URL linked by the "Search" link in the "Help" menu.
-#search_url = http://galaxyproject.org/search/usegalaxy/
+#search_url = http://galaxyproject.org/search/
```
Comment (nsoranzo) [inline config/galaxy.ini.sample]: https

File: config/galaxy.ini.sample [hunk 5]
```diff
@@ -892,8 +893,9 @@ use_interactive = True
 
 # -- Data Libraries
 
-# These library upload options are described in much more detail in the wiki:
-#   https://wiki.galaxyproject.org/Admin/DataLibraries/UploadingLibraryFiles
+# These library upload options are described in much more detail in the Galaxy
+# Community Hub:
+# https://galaxyproject.org/admin/data-libraries/uploading-library-files
```
Comment (nsoranzo) [inline config/galaxy.ini.sample]: https://galaxyproject.org/data-libraries/

File: config/galaxy.ini.sample [hunk 6]
```diff
@@ -1022,7 +1024,7 @@ use_interactive = True
 # users (email addresses).  These users will have access to the Admin section
 # of the server, and will have access to create users, groups, roles,
 # libraries, and more.  For more information, see:
-#   https://wiki.galaxyproject.org/Admin/Interface
+#   https://galaxyproject.org/admin/interface
```
Comment (nsoranzo) [inline config/galaxy.ini.sample]: https://galaxyproject.org/admin/

P.S.: https://wiki.galaxyproject.org/Admin/Interface is missing a proper redirect.

File: templates/webapps/galaxy/galaxy.masthead.mako [hunk 7]
```diff
@@ -15,12 +15,12 @@
             'lims_doc_url'              : app.config.get("lims_doc_url", "https://usegalaxy.org/u/rkchak/p/sts"),
             'biostar_url'               : app.config.biostar_url,
             'biostar_url_redirect'      : h.url_for( controller='biostar', action='biostar_redirect', qualified=True ),
-            'support_url'               : app.config.get("support_url", "https://wiki.galaxyproject.org/Support"),
+            'support_url'               : app.config.get("support_url", "https://galaxyproject.org/support"),
             'search_url'                : app.config.get("search_url", "http://galaxyproject.org/search/usegalaxy/"),
```
Comment (nsoranzo) [inline templates/webapps/galaxy/galaxy.masthead.mako]: https://galaxyproject.org/search/

File: templates/webapps/tool_shed/base_panels.mako [hunk 8]
```diff
@@ -115,13 +115,13 @@
                     if qa_url:
                         menu_options = [ [_('Galaxy Q&A'), qa_url, "_blank" ] ]
                     menu_options.extend( [
-                        [_('Tool Shed Wiki'), app.config.get( "wiki_url", "https://wiki.galaxyproject.org/ToolShed" ), "_blank" ],
-                        [_('Support'), app.config.get( "support_url", "https://wiki.galaxyproject.org/Support" ), "_blank" ],
+                        [_('Tool Shed Wiki'), app.config.get( "wiki_url", "https://galaxyproject.org/tool-shed" ), "_blank" ],
+                        [_('Support'), app.config.get( "support_url", "https://galaxyproject.org/support" ), "_blank" ],
                         [_('Search'), app.config.get( "search_url", "http://galaxyproject.org/search/usegalaxy/" ), "_blank" ],
```
Comment (nsoranzo) [inline templates/webapps/tool_shed/base_panels.mako]: https://galaxyproject.org/search/

File: lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py [hunk 9]
```diff
@@ -33,7 +33,7 @@ def get_message_for_no_shed_tool_config():
     message += 'tag that includes a tool_path attribute value which is a directory relative to the Galaxy installation directory in order to '
     message += 'automatically install tools from a tool shed into Galaxy (e.g., the file name shed_tool_conf.xml whose <toolbox> tag is '
     message += '<toolbox tool_path="../shed_tools">).  For details, see the "Installation of Galaxy tool shed repository tools into a local '
-    message += 'Galaxy instance" section of the Galaxy tool shed wiki at http://wiki.galaxyproject.org/InstallingRepositoriesToGalaxy#'
+    message += 'Galaxy instance" section of the Galaxy tool shed wiki at https://galaxyproject.org/installing-repositories-to-galaxy/'
     message += 'Installing_Galaxy_tool_shed_repository_tools_into_a_local_Galaxy_instance.'
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py]: The full link was continuing on the next line, so the proper link now is: https://galaxyproject.org/installing-repositories-to-galaxy/#installing-galaxy-tool-shed-repository-data-types-into-a-local-galaxy-instance

P.S.: http://wiki.galaxyproject.org/InstallingRepositoriesToGalaxy is missing a proper redirect.

File: templates/admin/tool_shed_repository/select_tool_panel_section.mako [hunk 10]
```diff
@@ -51,11 +51,11 @@
         repository tools may include code that produces malicious behavior, so be aware of what you are installing.
     </p>
     <p>
-        If you discover a repository that causes problems after installation, contact <a href="https://wiki.galaxyproject.org/Support" target="_blank">Galaxy support</a>,
+        If you discover a repository that causes problems after installation, contact <a href="https://wiki.galaxyproject.org/support" target="_blank">Galaxy support</a>,
```
Comment (nsoranzo) [inline templates/admin/tool_shed_repository/select_tool_panel_section.mako]: https://galaxyproject.org/support/

File: templates/webapps/galaxy/admin/center.mako [hunk 11]
```diff
@@ -104,7 +104,7 @@ Please visit <a href="https://wiki.galaxyproject.org/Admin" target="_blank">the
         <h4>Sample tracking</h4>
             <ul>
                 <li>
-                    Please see the <a href="https://wiki.galaxyproject.org/Admin/DataLibraries/LibrarySampleTracking" target="_blank">sample tracking tutorial</a>.
+                    Please see the <a href="https://galaxyproject.org/admin/data-libraries/library-sample-tracking" target="_blank">sample tracking tutorial</a>.
```
Comment (nsoranzo) [inline templates/webapps/galaxy/admin/center.mako]: https://galaxyproject.org/archive/library-sample-tracking/

P.S.: https://wiki.galaxyproject.org/Admin/DataLibraries/LibrarySampleTracking is missing a proper redirect.

File: templates/webapps/tool_shed/base_panels.mako [hunk 12]
```diff
@@ -115,13 +115,13 @@
                     if qa_url:
                         menu_options = [ [_('Galaxy Q&A'), qa_url, "_blank" ] ]
                     menu_options.extend( [
-                        [_('Tool Shed Wiki'), app.config.get( "wiki_url", "https://wiki.galaxyproject.org/ToolShed" ), "_blank" ],
-                        [_('Support'), app.config.get( "support_url", "https://wiki.galaxyproject.org/Support" ), "_blank" ],
+                        [_('Tool Shed Wiki'), app.config.get( "wiki_url", "https://galaxyproject.org/tool-shed" ), "_blank" ],
```
Comment (nsoranzo) [inline templates/webapps/tool_shed/base_panels.mako]: https://galaxyproject.org/toolshed/

**Review by nsoranzo — CHANGES_REQUESTED** [review]

File: lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py (hunk 9 above)
Comment (dannon) [reply lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py]: @nsoranzo This one's not quite right -- given the context it looks like it should just link to the base page.

File: templates/admin/tool_shed_repository/select_tool_panel_section.mako (hunk 10 above)
Comment (dannon) [reply templates/admin/tool_shed_repository/select_tool_panel_section.mako]: @nsoranzo What's wrong with this one?

Comment (dannon) [reply templates/admin/tool_shed_repository/select_tool_panel_section.mako]: Argh, nevermind :)

File: config/galaxy.ini.sample (hunk 6 above)
Comment (dannon) [reply config/galaxy.ini.sample]: https://github.com/galaxyproject/infrastructure-playbook/commit/32d21f9efdbd2ca03532d0fa38184fde4db21d89

File: templates/webapps/galaxy/admin/center.mako (hunk 11 above)
Comment (dannon) [reply templates/webapps/galaxy/admin/center.mako]: https://github.com/galaxyproject/infrastructure-playbook/commit/97fe8f28d0e9f91fd422db185706d1e39214c9e4

File: lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py (hunk 9 above)
Comment (dannon) [reply lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py]: https://github.com/galaxyproject/infrastructure-playbook/commit/97fe8f28d0e9f91fd422db185706d1e39214c9e4

Comment (nsoranzo) [reply lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py]: > @nsoranzo This one's not quite right -- given the context it looks like it should just link to the base page.

Agreed.

File: client/galaxy/scripts/mvc/library/library-folderlist-view.js [hunk 13]
```diff
@@ -386,7 +386,7 @@ var FolderListView = Backbone.View.extend({
       '</table>',
       '<div class="empty-folder-message" style="display:none;">',
           'This folder is either empty or you do not have proper access permissions to see the contents. If you expected something to show up',
-          ' please consult the <a href="https://galaxyproject.org/admin/data-libraries/library-security" target="_blank">library security wikipage</a>',
+          ' please consult the <a href="https://galaxyproject.org/admin/data-libraries/#permissions" target="_blank">library security wikipage</a>',
```
Comment (nsoranzo) [inline client/galaxy/scripts/mvc/library/library-folderlist-view.js]: https://galaxyproject.org/data-libraries/#permissions

File: config/galaxy.ini.sample [hunk 14]
```diff
@@ -895,7 +895,7 @@ use_interactive = True
 
 # These library upload options are described in much more detail in the Galaxy
 # Community Hub:
-# https://galaxyproject.org/admin/data-libraries/uploading-library-files
+# https://galaxyproject.org/admin/data-libraries/
```
Comment (nsoranzo) [inline config/galaxy.ini.sample]: https://galaxyproject.org/data-libraries/

**Review by nsoranzo — APPROVED** [review]
Thanks @dannon for bearing with me, there are still 2 small partially-fixed tweaks, I'll fix them, resolve the merge conflicts locally and merge.

**Comment by dannon** [comment]
@nsoranzo Sounds good, thanks!  Always feel free to just PR against my branches or whatever, too, if that's easier.

**Pull request merged**